lib/providers/events/app_opened_event_provider.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:Medito/constants/constants.dart';
+import 'package:Medito/models/models.dart';
+import 'package:Medito/providers/providers.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+
+var appOpenedEventProvider = FutureProvider<void>((ref) async {
+  var deviceInfo = await ref.read(deviceAndAppInfoProvider.future);
+  var data = _setAppOpenedModelData(deviceInfo);
+
+  await ref.read(eventsProvider(event: data.toJson()).future);
+});
+
+EventsModel _setAppOpenedModelData(DeviceAndAppInfoModel val) {
```

Comment:
Is this a setter or a getter?

Reply:
This is only for creating a model I changed the name to _createAppOpenedModelData